yodeploy/cmds/yodeploy_server.py

```diff
@@ -29,11 +30,42 @@ def parse_args(flask_app):
     return opts
 
 
+def create_context():
+    if hasattr(ssl, 'create_default_context'):  # Python >= 2.7.9
+        context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+        context.options |= ssl.OP_CIPHER_SERVER_PREFERENCE
+    else:
+        context = LegacySSLContext()
+
+    context.load_cert_chain(
+        certfile=flask_app.config.server.ssl.cert_chain,
+        keyfile=flask_app.config.server.ssl.key)
+    context.set_ciphers(flask_app.config.server.ssl.ciphers)
```

Comment:
Ah... so this is where https://github.com/yola/chef/pull/2095 is consumed (talking to self)

Reply:
We could have a default here of `HIGH` or something. But I was being lazy.